service/src/test/java/com/ge/predix/controller/test/PolicyEvaluationControllerIT.java

```diff
@@ -104,13 +105,39 @@ public void setup() {
         this.notApplicableAndDenyPolicySets = createNotApplicableAndDenyPolicySets();
     }
 
+    @BeforeMethod
```

Comment:
This beforeMethod() is redundant.  setup() will create test zone and setup mock context for all tests.  Then in testPolicyZoneDoesNotExistException() you can overwrite security context with null zone at the beginning of the test and restore it to the test zone at the end of the test.